docs/release_notes.md

```diff
@@ -8,6 +8,30 @@ We are tracking two types of major changes:
 
 If you are creating the release notes for a new version, please see the [template](#template-version-majorminorpatch). Release notes should capture all of the significant changes since the last numbered package release.
 
+# Version 1.1.0
+
+This version focuses on improving the clarity of the telemetry reported.
+
+## Users
+
+### User Improvements
+
+- Report splits `stages_completed` into stage status and duration.
+
+## User Breaking Changes
+
+None.
+
+## Developers
+
+### Developer Improvements
+
+None
+
+### Developer Breaking Changes
+
+None
+
```

Comment:
Do we really want to update this file every time the minor version changes? Looks like this could grow very quickly.